browser/js/collisions.js

```diff
@@ -0,0 +1,57 @@
+// Collisions
```

Comment:
moved collisions to a different file until later when we will use it. Still accessible.